### DISCUSSION TITLE
Time stamp correction and forced alignment

### DISCUSSION THREAD

**Discussion by phliulei** [SYSTRAN/faster-whisper#158] [Ideas]
After completing recognition, I need to proofread. At this point, I already have the correct text content, and if I want to adjust the timestamps of the words based on this text, that is, force alignment, is this possible?

**Comment by EtienneAb3d** [comment]
@phliulei, you may try this:
https://github.com/EtienneAb3d/WhisperTimeSync
;-)

**Reply by phliulei** [reply]
> @phliulei, you may try this:
> https://github.com/EtienneAb3d/WhisperTimeSync
> ;-)

Thank you so much
